test/components/beta/avatar_stack_test.rb

```diff
@@ -50,7 +50,7 @@ def test_renders_default_body
     end
 
     assert_selector(".AvatarStack") do
-      refute_selector(".toltipped")
+      refute_selector(".tooltipped")
```

Comment:
This is scary

Reply:
Hmm yeah would you mind reverting this commit? I think we'll have to handle it separately.